src/SqlProcessor.php

```diff
@@ -221,9 +221,10 @@ public function processModifier($type, $value)
 						return $this->processArray("any[]", $value);
 
 					case 'i[]':
-						$i = count($value);
-						while ($i-- && is_int($value[$i]));
```

Comment:
That would certainly work, but it would use the `processArray` fallback too often when it's not needed.

Would you prefer
```diff
  $i = count($value);
- while ($i-- && is_int($value[$i]));
+ foreach ($value as $v) {
+     if (is_int($v) $i--;
+ }
  if ($i >= 0) break; // fallback to processArray 
```
?